ansible/files/paper-config/plugins/FactoryMod/config.yml

```diff
@@ -10665,7 +10698,7 @@ recipes:
           ==: ItemMeta
           display-name: Compact Repair Kit
           lore:
-            - One kit will repair a compactor to full health.
+            - One kit will repair a Compactor to full health.
```

Comment:
I know this is for consistency's sake, but do you really want to commit yourself to handling tickets of everyone wanting replacements compactor repair kits? :)

Reply:
oh.. right....